pallets/nft/src/lib.rs

```diff
@@ -610,20 +616,21 @@ impl<T: Config> NFTTrait<T::RuntimeOrigin> for Module<T> {
 
 pub mod migration {
     use crate::sp_api_hidden_includes_decl_storage::hidden_include::IterableStorageDoubleMap;
-    use crate::{Config, NFTsInCollection, NumberOfNFTs};
+    use crate::{Config, NFTOwner};
     use frame_support::storage::StorageMap;
+    use pallet_portfolio::PortfolioNFT;
     use sp_runtime::runtime_logger::RuntimeLogger;
 
-    pub fn migrate_to_v1<T: Config>() {
+    pub fn migrate_to_v2<T: Config>() {
         RuntimeLogger::init();
-        log::info!(">>> Initializing NFTsInCollection Storage");
-        initialize_nfts_in_collection::<T>();
-        log::info!(">>> NFTsInCollection was successfully updated");
+        log::info!(">>> Initializing NFTOwner Storage");
+        initialize_nft_owner::<T>();
+        log::info!(">>> NFTOwner was successfully updated");
     }
 
-    fn initialize_nfts_in_collection<T: Config>() {
-        for (ticker, _, id_count) in NumberOfNFTs::iter() {
-            NFTsInCollection::mutate(ticker, |collection_count| *collection_count += id_count);
+    fn initialize_nft_owner<T: Config>() {
+        for (portfolio_id, (ticker, nft_id), _) in PortfolioNFT::iter() {
+            NFTOwner::insert((ticker, nft_id), portfolio_id.did);
```

Comment:
We can't remove this migration it hasn't run on staging/testnet/mainnet yet.  These two migrations can be merged into one.

Reply:
Also no need to bump the version, since v1 hasn't run on any of the chains yet.

Reply:
My mistake, I forgot this was not in the v6.0 release. I'll add it back